pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -1252,6 +1252,22 @@ def test_type_error_multiindex(self):
         result = dg["x", 0]
         tm.assert_series_equal(result, expected)
 
+    def test_tuple_string_column_names(self):
```

Comment:
Can you move this to `pandas/tests/indexing/multiindex/test_getitem.py`?

Reply:
Done @mroeschke